lib/mongo/index.js

```diff
@@ -118,6 +118,13 @@ class Mongo extends EventEmitter {
     return mongoQuery(col, method, query, null, options)
   }
 
+  async bulkWrite(collection, operations, options = {}) {
+    const col = this.db.collection(collection)
+    // operations should be an array
+    if (!col || !Array.isArray(operations)) throw new Error('MONGO_BULKWRITE_PARAMS')
+    return col.bulkWrite(operations, options)
```

Comment:
can you use mongoQuery here? so it is standard across? also you didnt write unit test

Reply:
yep, have updated the PR to use mongoQuery and add unit test.